src/main/webapp/help-name.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    (Optional) The unique name of the provisioned VDB within a group. If unset, a name is randomly generated
```

Comment:
I don't think the `within a group.` is needed? Not all VDBs are part of a group?

Reply:
yes this was a bit confusing, this message is the same in Terraform as well, https://registry.terraform.io/providers/delphix-integrations/delphix/latest/docs/resources/vdb#argument-reference 

we can change if this is not the case

Reply:
Based on my experience, this is not the case.